### PR TITLE
Feat: Add PackerLoad command

### LINKED ISSUE #301: Feature request: Add a command to load plugins
### Idea

It'll be nice to have a command that loads a specific plugin . For example :
```
:PackerLoad telescope.nvim
```
loads telescope plugin (when opt=true) with it's setup and config if the plugin isn't already loaded.

### Motivation
I have most of the plugin configuration done in packer config . If I wanted to load a plugin I can do packadd but that means the configs won't be loaded . It would have been great if packer handled that . Also I think I can skip requires and after with this 😄

### PR DESCRIPTION
**Pull request by shadmansaleh**
Packer load command loads the plugins provided in argument

This was really easy everything was already there . Thanks @wbthomason @akinsho 

I still need to add some docs :)

closes #301

### CODE REVIEW

File: lua/packer.lua [hunk 1]
```diff
@@ -446,6 +447,28 @@ packer.compile = function(output_path)
...
+-- @param plugins string String of space separated plugins names
+--                      intended for PackerLoad command
+packer.loader = function(plugins)
+  local plugin_list = vim.tbl_filter(function(v) return #v > 0 end,
+                                     vim.split(plugins, ' '))
+  require('packer.load')(plugin_list, {}, _G.packer_plugins)
+end
+
+-- Completion for unloaded plugins
+-- Intended to provide completion for PackerLpad command
```
Comment (akinsho) [inline lua/packer.lua:460]:
```suggestion
-- Intended to provide completion for PackerLoad command
```
Minor typo

**Review by akinsho — COMMENTED** [review]
This looks great, thanks for adding the completion as well that's a nice touch 💯

**Comment by shadmansaleh** [comment]
I think I'm done . ready for review :)

**Review by akinsho — APPROVED** [review]
Just tried this out locally and it works quite nicely 👍🏿

**Comment by akinsho** [comment]
@wbthomason are you alright with me merging this, a bit of a moot point to ask since if you read it you could just hit the button yourself 😆 , more of a nudge then since it looks good and has been sitting around a bit

**Comment by wbthomason** [comment]
Absolutely, sorry for not getting to this until now. I missed that the docs had been added.

Thanks for the nudge, and thanks @shadmansaleh for the feature!

**Pull request merged**